VirtualList/useThemeVirtualList.js

```diff
@@ -141,7 +147,7 @@ const useSpottable = (props, instances) => {
 		mutableRef.current.isScrolledBy5way = false;
 		mutableRef.current.isScrolledByJump = false;
 
-		if (nextIndex >= 0) {
+		if (nextIndex >= 0 && index >= 0) {
```

Comment:
I think we need this change on useEvent.js line 159 too.

Reply:
I got the reason why we cannot applying this on useEvent.js line 159.
the "if" statement structure is different with sandstone.